.github/workflows/testing.yml

```diff
@@ -630,3 +632,79 @@ jobs:
         if: always()
         run: |
           docker rm taichi_build_desktop -f
+
+  build_android_demos:
+    name: Build Android Demos
+    needs: check_files
+    runs-on: [self-hosted, Linux, cn]
+    timeout-minutes: 60
+    permissions:
+      packages: read
+      contents: read
+    env:
+      REDIS_HOST: 172.16.5.8
```

Comment:
any security concern? maybe hide via token/secret?

Reply:
No security concern here.
Unity license is baked into docker image now, so there's nothing sensitive in the redis, only Android buildbot addresses and locks.